reconcile/gql_definitions/common/namespaces.py

```diff
@@ -274,7 +274,7 @@ class Config:
         extra = Extra.forbid
 
 
-class NamespaceManagedResourceNamesV1(BaseModel):
+class ManagedResourceNamesV1(BaseModel):
```

Comment:
This was due to https://github.com/app-sre/qontract-schemas/pull/382/files renamed the schema and was merged before the schema change for this.